Gu.Analyzers.CodeFixes/Gu.Analyzers.CodeFixes.csproj

```diff
@@ -18,5 +18,11 @@
   <ItemGroup>
     <ProjectReference Include="..\Gu.Analyzers.Analyzers\Gu.Analyzers.Analyzers.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="MakeInternalFixProvider.cs">
```

Comment:
This looks weird, is it needed for anything?

Reply:
Nope. I think my visual studio just added it for no reason.